packages/core/src/Layout/Layout.ts

```diff
@@ -1,14 +1,29 @@
-const { v4: uuidv4 } = require('uuid');
+import { v4 as uuidv4 } from 'uuid';
 
-const { PageSources } = require('../Page/PageSources');
-const { NodeProcessor } = require('../html/NodeProcessor');
+import { PageSources } from '../Page/PageSources';
+import { NodeProcessor } from '../html/NodeProcessor';
 
-const logger = require('../utils/logger');
+import { error } from '../utils/logger';
+import VariableProcessor from '../variables/VariableProcessor';
 
 const LAYOUT_PAGE_BODY_VARIABLE = 'content';
 
 class Layout {
-  constructor(sourceFilePath, config) {
+  sourceFilePath: string;
+  config: any;
+
+  includedFiles: Set<string>;
+  layoutProcessed: string;
+  layoutPageBodyVariable: string;
+  layoutPageNavUuid: string;
+  headTop: any[];
+  headBottom: any[];
+  scriptBottom: any[];
+  layoutUserScriptsAndStyles: any[];
```

Comment:
Could we also avoid the `any[]` types here?

Reply:
Just curious, what's the conclusion for this?